testbench/grpc_server.py

```diff
@@ -27,6 +27,12 @@ class StorageServicer(storage_pb2_grpc.StorageServicer):
     def __init__(self, db):
         self.db = db
 
+    def CreateBucket(self, request, context):
```

Comment:
This is super clean; I didn't realize methods are implemented and passed to the gRPC code to service calls.